Fix typos in `cresources.md` (#618)
It4innovations/hyperqueue: docs/jobs/cresources.md

## NUMA allocation strategy

> Workers automatically detect the number of CPUs and on Linux systems it also detects their partitioning into sockets.

Reviewer: ```suggestion
Workers automatically detect the number of CPUs and on Linux systems they also detect their partitioning into sockets.
```

> When the NUMA architecture is automatically detected, indexed resource with groups is used for resource "cpus".

Reviewer: ```suggestion
When a NUMA architecture is automatically detected, indexed resource with groups is used for resource "cpus".
```
